doc/sources/installation/installation-windows.rst

```diff
@@ -209,17 +210,25 @@ kivy with git rather than a wheel there are some additional steps:
    #. Install MinGW with::
 
         python -m pip install -i https://pypi.anaconda.org/carlkl/simple mingwpy
-
-   For Python 3.5 we use the MSVC compiler. For 3.5,
-   `Visual Studio 2015 <https://www.visualstudio.com/downloads/>`_ is
-   required, which is availible for free. Just download and install it and
-   you'll be good to go. 
+        
+   **MSVC (Python >= 3.5):**
+    
+   #. Download and install
+      `Visual Studio 2015 <https://www.visualstudio.com/downloads/>`_ .
+      Alternatively, use the smaller,
+      `Visual C Build Tools instead
+      <https://github.com/kivy/kivy/wiki/Using-Visual-C---Build-Tools-instead-of-Visual-Studio-on-Windows>`_.
+
+#. Set the environment variables.
+
+   .. warning::
    
-   Visual Studio is very big so you can also use the smaller,
-   `Visual C Build Tools instead
-   <https://github.com/kivy/kivy/wiki/Using-Visual-C---Build-Tools-instead-of-Visual-Studio-on-Windows>`_.
+      If compiling without the optional GSTREAMER dependency, set
+      ``USE_GSTREAMER=0`` instead of ``USE_GSTREAMER=1``.
```

Comment:
Which bit is unnecessary? The docs immediately above that say to compile with ``USE_GSTREAMER=1``, which results in a build error; see [here](https://github.com/kivy/kivy/issues/4941#issuecomment-274259328).